dev/com.ibm.ws.concurrent.persistent_fat_multiple/publish/servers/com.ibm.ws.concurrent.persistent.fat.multiple/server.xml

```diff
@@ -20,24 +20,42 @@
   <include location="../fatTestPorts.xml"/>
 
   <persistentExecutor id="executor1" jndiName="concurrent/executor1" pollInterval="4s" retryLimit="1" pollSize="100"/>
-
   <persistentExecutor id="executor2" jndiName="concurrent/executor2" taskStoreRef="defaultDatabaseStore" pollInterval="4s" retryLimit="1"/>
 
   <databaseStore id="defaultDatabaseStore" keyGenerationStrategy="SEQUENCE"/>
+  
+  <library id="FATJDBCLib">
+    <fileset dir="${shared.resource.dir}/derby" includes="derby.jar"/>
+  </library>
+  
+  <library id="AnonymousJDBCLib">
+    <fileset dir="${shared.resource.dir}/jdbc" includes="driver.jar"/>
+  </library>
 
   <!-- database for scheduled tasks, also accessible to the application -->
-  <dataSource id="DefaultDataSource" fat.modify="true" containerAuthDataRef="authx">
+  <dataSource id="DefaultDataSource" jndiName="jdbc/DefaultDataSource" containerAuthDataRef="authx">
     <jdbcDriver libraryRef="FATJDBCLib"/>
     <properties.derby.embedded createDatabase="create" databaseName="memory:persistmultidb"/>
   </dataSource>
-
-  <library id="FATJDBCLib">
-    <fileset dir="${shared.resource.dir}/derby" includes="derby.jar"/>
-  </library>
+  
+    <!-- anonymous database for scheduled tasks, also accessible to the application -->
+  <dataSource id="AnonymousDataSource" jndiName="jdbc/AnonymousDataSource">
+    <jdbcDriver libraryRef="AnonymousJDBCLib"/>
+    <properties serverName="${env.DB_HOST}" portNumber="${env.DB_PORT}" databaseName="${env.DB_NAME}" 
+    	user="${env.DB_USER}" password="${env.DB_PASS}" URL="${env.DB_URL}"/>
```

Comment:
typically configurations will configure as individual properties (serverName, portNumber, databasename, etc) OR using the URL, but not both.
Behavior is undefined if both individual props and URL is used. For easy portability across the DB types, I recommend just using the URL property, which is easily obtainable by calling `container.getJdbcUrl()`, and we can also override the dummy Derby container to return something like `dbc:derby:memory:persistmultidb;create=true`. This way we won't have to treat Derby Embedded any differently than the other DBs in the FAT setup code